ethcore/src/executive.rs

```diff
@@ -1136,16 +1144,50 @@ impl<'a, B: 'a + StateBackend> Executive<'a, B> {
         T: Tracer,
         V: VMTracer,
     {
+        let schedule = self.schedule;
+
+        //check if particualar transaction type is enabled at this block number in schedule
+        match t.as_unsigned() {
+            TypedTransaction::AccessList(_) => {
+                if !(schedule.eip2929 && schedule.eip2930) {
```

Comment:
```suggestion
                if !schedule.eip2930 {
```

In general, I do not see in other parts of the code checking EIP dependencies, so proballly it will consistent only checking for `eip2930`.

Reply:
changed to eip2930 only